saleor/graphql/checkout/tests/test_checkout.py

```diff
@@ -268,6 +268,36 @@ def test_checkout_create_with_unavailable_variant(
     assert error["variants"] == [variant_id]
 
 
+def test_checkout_create_with_malicious_variant_id(
+    api_client, stock, graphql_address_data, channel_USD
+):
+
+    variant = stock.product_variant
+    variant.channel_listings.filter(channel=channel_USD).update(price_amount=None)
+    test_email = "test@example.com"
+    shipping_address = graphql_address_data
+    variant_id = (
+        "UHJvZHVjdFZhcmlhbnQ6NDkxMyd8fERCTVNfUElQRS5SRUNFSVZFX01FU1N"
+        "BR0UoQ0hSKDk4KXx8Q0hSKDk4KXx8Q0hSKDk4KSwxNSl8fCc="
+    )
```

Comment:
I would add some comment to explain this ID - what is it and why it's malicious.